tox.ini

```diff
@@ -16,7 +16,7 @@ commands =
 [testenv:black]
 basepython = python3
 skip_install = true
-deps = black==22.3.0
+deps = black==24.4.2
 commands = black --check src test scripts {posargs}
```

Comment:
This PR + adding pre-commit has become a bit muddled.

The only reason these formatting changes have crept in, is because pre-commit config has the latest black. If you change the pre-commit config to use 22.3.0 everything should have checked just fine. So this PR is really "bump black version to the latest and reformat"

I'm keen to have pre-commit as part of the repo, but don't want to have to maintain two sets of config. We could change the black environment in tox to run pre-commit. Then the workflow can stay the same. Alternatively we could change the workflow definition to run pre-commit directly.
https://github.com/PyFixate/Fixate/blob/master/.github/workflows/test.yml#L37